src/instance/functions/style.js

```diff
@@ -13,13 +13,13 @@ export default function style (element) {
 	const inline = {}
 
 	const inlineStyle = element.node.getAttribute('style') || ''
-	const inlineMatch = inlineStyle.match(/.+[^;]/g)
+	const inlineMatch = inlineStyle.match(/([\w|:|\-\s|\.]+)/ig) || []
```

Comment:
I like this, ensuring `inlineMatch` is an array with or without a match.

Reply:
So for example, `-webkit-transform: rotate(30deg)` is not matched by that regex.

Using the wonderful [RegExr](www.regexr.com), I was able to piece together this one that seems to match inline CSS quite well:

```regex
/[\w-]+\s*:\s*[^;]+\s*/ig
```

I appreciate the contribution though, the case insensitivity alone was a great fix.

Reply:
Ahh good stuff mate, yeah your regex does seem a lot more solid than mine!